samwise.js

```diff
@@ -3,23 +3,24 @@
  *
```

Comment:
SCRIPT-This logic might be clearer if you used a switch statement instead of multiple if-else statements.